components/ContributorsList.vue

```diff
@@ -6,25 +6,24 @@
     <v-row>
       <v-col sm="4" md="5" lg="6" v-for="lead in leads" :key="lead.name">
         <contributor-lead
-        :githubHandle="lead.githubHandle"
-        :twitterHandle="lead.twitterHandle"
-        :name="lead.name"
-        :role="lead.role"
-        :customPic="lead.customPic" />
+          :githubHandle="lead.githubHandle"
```

Comment:
Same here. The indentation is off in the original for sure, but unless this is done automagically we should keep it separate.